skbio/io/format/stockholm.py

```diff
@@ -151,6 +151,39 @@
            }
        }
 
+.. note:: References labelled with ``RN``/``RM``/``RT``/``RA``/``RL``/``RC``
+   are handled differently than other GF features. When reading a Stockholm
+   file with these features, the reader populates adds a dictionary to a list
+   of reference dictionaries, and populates it with the information on that
+   specific reference. If the reference does not include all the expected
+   information (title, author, etc.), the dictionary will only be populated
+   with the information provided.
+
+   A single reference will be stored as::
+
+        metadata = {
+            'RN': [{
+                'RM': 'reference medline'
```

Comment:
When you convert this example into the multi-reference example below, add trailing commas to each key/value pair so it's valid Python, e.g. `'RM': 'reference medline',`